stripecardscan/src/main/java/com/stripe/android/stripecardscan/framework/api/dto/CardImageVerificationDetails.kt

```diff
@@ -10,11 +10,79 @@ internal data class CardImageVerificationDetailsRequest(
 
 @Serializable
 internal data class CardImageVerificationDetailsResult(
-    @SerialName("expected_card") val expectedCard: CardImageVerificationDetailsResultCard?
+    @SerialName("expected_card")
+    val expectedCard: CardImageVerificationDetailsExpectedCard?,
+
+    @SerialName("accepted_image_configs")
+    val acceptedImageConfigs: CardImageVerificationDetailsAcceptedImageConfigs? = null,
 )
 
 @Serializable
-internal data class CardImageVerificationDetailsResultCard(
+internal data class CardImageVerificationDetailsExpectedCard(
     @SerialName("issuer") val issuer: String?,
     @SerialName("last4") val lastFour: String?,
 )
+
+@Serializable
+internal enum class CardImageVerificationDetailsFormat {
+    @SerialName("heic")
+    HEIC,
+
+    @SerialName("jpeg")
+    JPEG,
+
+    @SerialName("webp")
+    WEBP;
+}
+
+@Serializable
+internal data class CardImageVerificationDetailsImageSettings(
+    @SerialName("compression_ratio") var compressionRatio: Double? = null,
+    @SerialName("image_size") var imageSize: DoubleArray? = null,
+) {
+    companion object {
+        // These default values are what Android was using before the addition of a server config.
+        val DEFAULT = CardImageVerificationDetailsImageSettings(
+            0.92,
+            doubleArrayOf(1080.0, 1920.0)
```

Comment:
nits:

1. name parameters when on new lines
2. I'm guessing this has to be an array for the DTO. Can we create a business-logic domain object that uses `Size` instead of an array to be explicit about the contents?

Alternatively, can we use a `typealias` here to make it clear that this is a WxH array? see `RectForm` for an example.